roles/openshift_prometheus/vars/main.yaml

```diff
@@ -1,2 +1,4 @@
 ---
 # vars file for openshift_prometheus
+openshift_prometheus_namespace: prometheus
```

Comment:
Probably should be something like 'openshift-prometheus' or something... @smarterclayton ??